cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/recommend-config.js

```diff
@@ -259,10 +258,10 @@ function transformItem(item) {
         // 3. Get the variant price
         if (selectedVariant) {
             if (selectedVariant.promotionalPrice && selectedVariant.promotionalPrice[algoliaData.currencyCode] !== null) {
-                item.promotionalPrice = selectedVariant.promotionalPrice[algoliaData.currencyCode];
+                item.promotionaPrice = selectedVariant.promotionalPrice[algoliaData.currencyCode];
```

Comment:
This is never used anywhere, and the display of `item.promotionalPrice` will be broken.
You need to do a similar thing than you did with `displayPrice`